flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionVertex.java

```diff
@@ -632,8 +632,8 @@ void executionFailed(Throwable t) {
 	/**
 	 * Simply forward this notification. This is for logs and event archivers.
 	 */
-	void notifyStateTransition(ExecutionAttemptID executionId, ExecutionState newState, Throwable error) {
-		getExecutionGraph().notifyExecutionChange(getJobvertexId(), subTaskIndex, executionId, newState, error);
+	void notifyStateTransition(ExecutionAttemptID executionId, ExecutionState newState, Throwable error, long errorTimestamp) {
```

Comment:
It's not strictly speaking the error timestamp, right? It's more of a state transition timestamp.